source/reference/minio-mc/mc-ilm-rule-ls.rst

```diff
@@ -25,6 +26,8 @@ The :mc:`mc ilm rule ls` command summarizes all configured object lifecycle mana
 
 .. end-mc-ilm-rule-ls-desc
 
+The :mc:`mc ilm rule list` alias has equivalent functionality to :mc:`mc ilm rule ls`.
```

Comment:
Swap `alias` here for `command`.
Probably need to do that throughout for all the affected commands where you use this phrasing.

Reply:
Done